docs/root/configuration/http/http_filters/compressor_filter.rst

```diff
@@ -31,7 +31,7 @@ An example configuration of the filter may look like the following:
 .. code-block:: yaml
 
     http_filters:
-    - name: compressor
+    - name: envoy.filters.http.compressor
```

Comment:
Note that this isn't explicitly required (the config will load based on typed_config below), but I think this is a fine change to reduce confusion.